### ISSUE TITLE
SQLite regression bug after Liquibase 4.8.0 release. Generated sql syntax errors for addAutoIncrement, addDefaultValue, dropColumn, mergeColumns

### ISSUE THREAD

**Issue by AlexanderSashchenko** [liquibase/liquibase#2644]
## Environment

**Liquibase Version**: 4.8.0

**Liquibase Integration & Version**: CLI, test-harness(Command Framework)

**Liquibase Extension(s) & Version**: na

**Database Vendor & Version**: sqlite 3.34.0

**Operating System Type & Version**: windows 10

## Description

After the release of Liquibase 4.8.0 a bug was found. This bug doesn't get reproduced on previous versions. The bug is reproduced in case you try to use Liquibase 'update' command against SQLite database (version 3.34.0 in my case) for the change types:
_- addAutoIncrement;
- addDefaultValue (also applicable for attributes defaultValueBoolean, defaultValueDate, defaultValueNumeric in case you use it);
- dropColumn
- mergeColumns_
The bug is about generating wrong SQL queries for named above change types. For _addAutoIncrement, addDefaultValue and dropColumn_ Liquibase generates "CREATE TABLE table_name ()" query. For _mergeColumns_ it generates "UPDATE full_name_table SET full_name = first_name || ' ' || last_name". Examples of used changelogs and full exception logs are attached.

## Steps To Reproduce

Run Liquibase 'update' command on version 4.8.0 against SQLite database for named above change types.

## Actual Behavior
SQL exceptions described in description section get thrown. Detailed logs are attached (.xml files are saved as .txt files).

## Expected/Desired Behavior
Successful deployment of named change types
[dropColumn full exception log.txt](https://github.com/liquibase/liquibase/files/8261654/dropColumn.full.exception.log.txt)
[dropColumn.txt](https://github.com/liquibase/liquibase/files/8261655/dropColumn.txt)
[addDefaultValue full exception log.txt](https://github.com/liquibase/liquibase/files/8261656/addDefaultValue.full.exception.log.txt)
[addDefaultValue.txt](https://github.com/liquibase/liquibase/files/8261657/addDefaultValue.txt)
[mergeColumns full exception log.txt](https://github.com/liquibase/liquibase/files/8261689/mergeColumns.full.exception.log.txt)
[mergeColumns.txt](https://github.com/liquibase/liquibase/files/8261690/mergeColumns.txt)
[addAutoIncrement full exception log.txt](https://github.com/liquibase/liquibase/files/8261686/addAutoIncrement.full.exception.log.txt)
[addAutoIncrement.txt](https://github.com/liquibase/liquibase/files/8261688/addAutoIncrement.txt)

**Comment by kataggart** [comment]
@AlexanderSashchenko thanks for all the detail here; we will definitely take a look

**Comment by bamthomas** [comment]
Any news on this issue ? 
(we are having the same problem, and dependabot asks us to use the 4.8.0)

**Comment by MalloD12** [comment]
I was not able to reproduce this issue on below environment:

```
Liquibase Version: 4.15.0

Liquibase Integration & Version: CLI

Liquibase Extension(s) & Version: N/A

Database Vendor & Version: Sqlite 3.36.0 

Operating System Type & Version: Mac OS (M1)
```

I tried using examples (changelogs) provided here and all of them have been executed successfully.

Closing this issue by now. @Jerbell could you please try again on a recent version and please let me know if you are still able to reproduce it? 

Thanks,
Daniel.

**Comment by Jerbell** [comment]
> Closing this issue by now. @Jerbell could you please try again on a recent version and please let me know if you are still able to reproduce it?

Hi @MalloD12
I don't see an issue that I reported related to this one, so I'm not sure what to test.  I also don't use Mac OS.

**Comment by MalloD12** [comment]
> > Closing this issue by now. @Jerbell could you please try again on a recent version and please let me know if you are still able to reproduce it?
> 
> Hi @MalloD12 I don't see an issue that I reported related to this one, so I'm not sure what to test. I also don't use Mac OS.

My apologies  @Jerbell I did not mean to tag you on this one. Please ignore my comment.

**Comment by maloewe-ona** [comment]
For future reference, it looks like this bug might have been introduced by pull request #2398, and was fixed for 4.11.0 by pull request #2778.

I was investigating this some time ago, and if I remember it correctly, Liquibase is looking up the tables at some point during changelog execution, but due to the broken table name escaping it wasn't finding the tables and then in subsequent steps used an empty string as table name. But I might be misremembering this.
